src/app/admin/general-settings/general-settings.component.html

```
@@ -119,6 +119,25 @@ <h3 class="title-small mb-2 mt-6">Brand assets</h3>
           placeholder="https://section.example.org/logo.svg"
         />
       </mat-form-field>
+      <input
+        #logoAssetInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)